templates/app_layout.html

```diff
@@ -17,7 +17,10 @@
   </head>
   <body>
     <nav class="navbar navbar-expand-sm">
-      <div class="collapse navbar-collapse d-flex justify-content-between" id="collapsibleNavbar">
+      <div
+        class="collapse navbar-collapse d-flex justify-content-between"
+        id="collapsibleNavbar"
+      >
         <ul class="navbar-nav">
           <li>
             <a class="nav-link mr-4 app-title" href="/">AcadeMe</a>
```

Comment:
Just a regular CSS file used to style some of the containers and components in the website
no need to panic 🐼